src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,22 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
 
+@Entity
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany
+    @Cascade(CascadeType.MERGE)
```

Comment:
if we should use existing smiles from db, merge is not necessary here, we don't need cascade types for this relation